### DISCUSSION TITLE
How to display TimePicker only for Hour and Minute only ?

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#6373] [Help] [answered]
How to display TimePicker only for Hour and Minute only ?

![image](https://user-images.githubusercontent.com/6214964/235498701-61491b21-f996-4040-b761-ac2ec2cd0c80.png)

I tried ->displayFormat('H:i')  and ->format('H:i') but not work.

Thank you

**Comment by danharrin** [comment]
`->disableSeconds()`

**Reply by jigujigu** [reply]
it's show me error : Method Filament\Forms\Components\TimePicker::disableSeconds does not exist.

**Reply by howdu** [reply, marked as answer]
Try `withoutSeconds`

**Reply by marcelloinfoweb** [reply]
It also doesn't exist

**Reply by danharrin** [reply]
`seconds(false)` ?
